Ports/.port_include.sh

```diff
@@ -407,10 +407,10 @@ func_defined patch_internal || patch_internal() {
     # patch if it was not yet patched (applying patches multiple times doesn't work!)
     if [ -z "${IN_SERENITY_PORT_DEV:-}" ] && [ -d patches ]; then
         for filepath in patches/*.patch; do
-            filename=$(basename $filepath)
-            if [ ! -f "$workdir"/.${filename}_applied ]; then
-                run patch -p"$patchlevel" < "$filepath"
-                run touch .${filename}_applied
+            filename="$(basename \"${filepath}\")"
+            if [ ! -f "${workdir}/.${filename}_applied" ]; then
+                run git apply --directory="Ports/${port}/${workdir}" -vp"${patchlevel}" < "${filepath}"
```

Comment:
Do we really want `-v` here?

Reply:
I found during testing that this command can fail silently, so perhaps we should exit gracefully when that happens.

The output is a bit verbose so I won't make it the default, but it can be useful when updating patches.